toolbox/certbot/cron_entry

```diff
@@ -0,0 +1 @@
+0 */12 * * * root /usr/airflow/atd-airflow/toolbox/certbot/renew.sh >> /var/log/airflow_cert_renewal.log
```

Comment:
Normally, I'd think maybe we should setup a `logrotate.d` file also to roll this log, but I think the amount of data we'll accumulate in there is going to be plenty small to not worry about it. Do you agree?

Reply:
i say we rotate it! set it and forget it!